Frameworks/CoreGraphics/CGColor.mm

```diff
@@ -62,11 +62,9 @@ CGColorRef CGColorCreateCopy(CGColorRef color) {
  @Status Interoperable
 */
 CGColorRef CGColorCreateCopyWithAlpha(CGColorRef color, float alpha) {
-    ColorQuad curColor;
+    const ColorQuad* curColor = [(UIColor*)color _getColors];
 
-    [(UIColor*)color getColors:&curColor];
-
-    return static_cast<CGColorRef>([[_LazyUIColor colorWithRed:curColor.r green:curColor.g blue:curColor.b alpha:alpha] retain]);
+    return static_cast<CGColorRef>([[_LazyUIColor colorWithRed:curColor->r green:curColor->g blue:curColor->b alpha:alpha] retain]);
```

Comment:
Understood, this will be part of the rework we do for CoreGraphics in the near future.

---
In reply to: [71058697](https://github.com/Microsoft/WinObjC/pull/658#discussion_r71058697) [](ancestors = 71058697)